bin/deps.ml

```diff
@@ -0,0 +1,40 @@
+(*
+ * Copyright (c) 2020 Ulysse Gérard <ulysse@tarides.com>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *)
+
+let run (`Setup ()) (`Syntax syntax) (`File file) =
+  let syntax = match syntax, Mdx.Syntax.infer ~file with
+    | Some s, _
+    | None, Some s -> s
+    | None, None -> Printf.eprintf
+      "Fatal error: could not infer syntax from filename %s, use the --syntax \
+      option to specify a syntax.\n" file; exit 1
+    in
+    let doc = Mdx.parse_file
+      syntax
+      file
+    in
+    let deps = Mdx.Dep.of_lines doc in
+    let deps = List.map Mdx.Dep.to_string deps in
+    let deps = String.concat " " deps in
+    Printf.printf "%s\n" deps;
```

Comment:
Now that we can only pass one file at a time, I think it would be better to separate the outputs with newlines instead of spaces.
Unlike spaces, newlines are very uncommon in file names and are expected to break every tools that print or read file names.

Alternatively, since this is meant to be consumed by dune only, we could use a sane format (eg. sexp or json).